### PR TITLE
Update readme for instructions on previewing the site locally

### PR DESCRIPTION
**Pull request by melcorr**
* Thanks to ekohl

Cherry-pick into:

* [ ] Foreman 3.0
* [ ] Foreman 2.5 (Satellite 6.10)
* [ ] Foreman 2.4
* [ ] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

File: web/README.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ Install `hugo` utility.
 
 ## Building
 
+While the `make` commands outliined in the [guides readme](https://github.com/theforeman/foreman-documentation/blob/master/guides/README.md) output previous of the guides, for a local preview of the full site, and links, do the following:
```
Comment (ekohl) [inline web/README.md:11]:
```suggestion
While the `make` commands outlined in the [guides readme](https://github.com/theforeman/foreman-documentation/blob/master/guides/README.md) output previous of the guides, for a local preview of the full site, and links, do the following:
```

**Review by ekohl — COMMENTED** [review]
There is a `Makefile` in the `web` directory but I'm not sure how useful it is.

You may also want to mention that in Fedora you need to run `dnf install hugo`.

File: web/README.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ Landing page for [docs.theforeman.org](https://docs.theforeman.org) is available
 
 Install `hugo` utility.
 
+On a Fedora system:
```
Comment (ekohl) [inline web/README.md:9]: How about shortening it a bit:
```suggestion
Install `hugo`, for example on a Fedora system:
```

File: web/README.md [hunk 3]
```diff
@@ -14,6 +20,9 @@ While the `make` commands outlined in the [guides readme](https://github.com/the
 $ cd web
 $ hugo server
 ```
+
+Alternatively, you can use `make server`
```
Comment (ekohl) [inline web/README.md:24]: I realized there is one trick that makes it easier: you can run `make -C web server` from the root. Not sure if we want to document this.

Comment (melcorr) [reply web/README.md:24]: I am worried about the LCD of contributors, so would avoid.

**Review by ekohl — APPROVED** [review]

**Comment by melcorr** [comment]
Thanks for your suggestions!

**Pull request merged**